style.css

```diff
@@ -101,3 +102,42 @@ body {
   opacity: 1;
   width: 100%;
 }
+
+.main {
+  padding-right: 0 63% 100px 0;
+  background: top right no-repeat url(img/mainImg.svg);
+}
+
+.main__title {
+  margin: 0 0 40px 0;
+  font: bold 50px/65px "pt_serif", arial, helvetica, sans-serif;
+}
+
+.main__subtitle {
+  margin: 0 0 41px 0;
+  font: 500 18px/21px "Roboto", arial, helvetica, sans-serif;
+}
+
+.main__button {
+  display: inline-block;
+  font: 500 14px/16px "Roboto", arial, helvetica, sans-serif;
+  color: #fff;
+  background: #0056ec;
+  padding: 14px 16px;
+  text-decoration: none;
+  border-radius: 4px;
+  transition: opacity .5s;
+}
+
+.main__button:hover {
+opacity: .5;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/new_project2.0/pullRequest?prid=6274500)